charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -103,4 +103,11 @@ spec:
             for: 15m
             labels:
               severity: warning
+          - alert: VeleroBackupOlderThan30Hours
```

Comment:
What about backups that should only be run every week? Or backups that should run every 6 hours?

I don't think it makes sense to have a global alert for this, as every backup is different.

While looking up the specifics of this, I couldn't find this metric? Who exposes it?